WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormActionsFactory.swift

```diff
@@ -183,8 +183,8 @@ private extension ProductFormActionsFactory {
             // The price settings action is always visible in the settings section.
             return true
         case .reviews:
-            // The reviews action is always visible in the settings section.
-            return true
+            // The reviews action is visible only if reviews are enabled in product settings.
+            return product.reviewsAllowed
```

Comment:
Maybe this condition `product.reviewsAllowed` could be in the function that determines the actions for a product type?

For example, this line:
https://github.com/woocommerce/woocommerce-ios/blob/eecf8d7b0c505402b1adea3c5527bfa1da62bc85/WooCommerce/Classes/ViewRelated/Products/Edit%20Product/ProductFormActionsFactory.swift#L82
could be
`let shouldShowReviewsRow = isEditProductsRelease3Enabled && product.reviewsAllowed`

Generally, this `isVisibleInSettingsSection` is meant for determining if an action should appear in the "settings section" instead of the bottom sheet. It still works in this case because [`.reviews` isn't supported in the bottom sheet](https://github.com/woocommerce/woocommerce-ios/blob/eecf8d7b0c505402b1adea3c5527bfa1da62bc85/WooCommerce/Classes/ViewRelated/Products/Edit%20Product/BottomSheetListSelector/ProductFormBottomSheetAction.swift#L5-L31).

I'd also suggest adding unit tests for this change 🙂

Reply:
Thank you Jaclyn for this super useful suggestion! I updated the code here aa44a288a1f429321adff2b6665146b170e6f768 and I added a test case for when reviews are disabled. Can you take a second look at it please?

Reply:
The changes look good to me! thanks Paolo 😄